### DISCUSSION TITLE
Freetext Notes within each blockset

### DISCUSSION THREAD

**Discussion by Jonathan-B-Reichel** [proginosko/LeechBlockNG#491] [Ideas]
It would be nice to have a free-text field that would allow notes, comments, or a detailed description of what each block set is accomplishing.  I have multiple block sets, and it would be nice to write out what motivates each one of them, so that it is easier to remember how/ why to make changes at some arbitrary point in the future.

**Comment by proginosko** [comment]
You can enter comments into the list of sites by prefixing with the '#' character. The only limitation is that you can't use spaces, so use underscores instead.

**Comment by smayer97** [comment]
Or any other space-holding character, like a period, '.'

**Comment by Vasdranna** [comment]
Could the automatic sorting be disabled?
Because I find pretty much useless to have the comments gone into oblivion, completely detatched from the lines they were referring to. Couldn't the domain sites field stay as it is after saving?

**Reply by proginosko** [reply]
Yes, I can add this as an option.

**Reply by proginosko** [reply]
Done. Option will appear in next release.

**Reply by Vasdranna** [reply]
Thank you so much, I didn't expect such a fast solution!

**Reply by proginosko** [reply]
Honestly, neither did I! 😂
